Modules/vc-module-export/VirtoCommerce.ExportModule.Core/Model/ExportDataQuery.cs

```diff
@@ -11,7 +11,7 @@ namespace VirtoCommerce.ExportModule.Core.Model
     public abstract class ExportDataQuery : ValueObject
     {
         /// <summary>
-        /// This used to instantiate data query of this type at export start.
+        /// This used to instantiate a data query of this type at export start.
```

Comment:
"ExportDataQuery type name is used to properly deserialize ExportDataQuery in backrgound task performing export"